__tests__/app.test.js

```diff
@@ -81,3 +81,32 @@ describe("GET /api/reviews/:review_id", () => {
             });
     });
 });
+
+describe("GET /api/reviews", () => {
+    test("200: Responds with an array of review objects", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                reviews.forEach((review) => {
+                    expect(review).toMatchObject({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        review_body: expect.any(String),
```

Comment:
We shouldn't be returning the review_body here. Easy to miss on the ticket, but we only want the review body when making a GET request for an individual review.